lib/spack/spack/build_systems/intel.py

```diff
@@ -24,11 +24,44 @@
 ##############################################################################
 
 import os
+import sys
+import glob
+import re
+import inspect
 import xml.etree.ElementTree as ET
+import llnl.util.tty as tty
 
-from llnl.util.filesystem import install, join_path
-from spack.package import PackageBase, run_after
+from llnl.util.filesystem import \
+    install, join_path, ancestor, \
+    LibraryList, find_headers, find_libraries, find_system_libraries
+
+from spack.version import Version
+from spack.package import PackageBase, run_after, InstallError
 from spack.util.executable import Executable
+from spack.util.prefix import Prefix
+from spack.build_environment import dso_suffix
+from spack.environment import EnvironmentModifications
+
+
+# A couple of utility functions that might be useful in general. If so, they
+# should really be defined elsewhere, unless deemed heretical.
+# (Or na"ive on my part).
```

Comment:
Probably should be defined elsewhere, but I'm not sure where. @tgamblin or @alalazo would know better.